lambdas/services/search_patient_details_service.py

```diff
@@ -26,8 +25,7 @@ def __init__(self, user_role, user_ods_code):
 
     def handle_search_patient_request(self, nhs_number):
         try:
-            auth_service = NhsOauthService(self.ssm_service)
-            pds_api_service = get_pds_service()(self.ssm_service, auth_service)
+            pds_api_service = get_pds_service()
```

Comment:
For the sake of consistency, could we modify the name here to `pds_service` as set in other parts of the code? (e.g lloyd_george_validator.py line 215)
```suggestion
            pds_service = get_pds_service()
```

Reply:
Good shout! I like your thinking, `pds_api_service` is a pretty loaded term, being the name of the unstubbed service.